benchmarks/benchmark/tools/locust-load-inference/README.md

```diff
@@ -51,13 +53,13 @@ You will set the `gcs_path` in your `terraform.tfvars` to this gcs path containi
 
 ### Step 2: create and give service account access to view dataset
 
-The Locust workload requires storage.admin access to view the dataset in the given gcs bucket. If you are running with workload identity, it obtains this access via a kubernetes service account that is backed by a gcloud service account. If you followed steps in `../../infra`, then you already have a kubernetes and gcloud service account created that you can use here.
+The Locust workload requires storage.admin access to view the dataset in the given gcs bucket. If you are running with workload identity, it obtains this access via a kubernetes service account that is backed by a gcloud service account. If you followed steps in `/infra`, then you already have a kubernetes and gcloud service account created that you can use here.
```

Comment:
`../../infra` is the correct location right? Why change it?

Reply:
we utilized "infra/" across the other locations so this change is to make it consistent - I can change those to use the relative path as desired.

Reply:
I updated all of them to ../../infra instead :)